iset.mm

```diff
@@ -135016,6 +135038,642 @@ the base set to the (extended) reals and which is nonnegative, symmetric
+      cxms sqxpeqd reseq2d fveq2d eqeq12d anbi12d isxms 3bitr4g ) ACIJZCKLZCMLZ
+      CNLZUNOZPZQLZRZUADIJZDKLZDMLZDNLZVBOZPZQLZRZUACUCJDUCJAUKUSURVFACDABUNVBE
+      FSHUBAULUTUQVEHAUPVDQAVABBOZPZUPVDAUMVGPVHUPGAVGUOUMABUNEUDUESAVGVCVAABVB
+      FUDUESUFUGUHUPULCUNULTUNTUPTUIVDUTDVBUTTVBTVDTUIUJ $.
+
+    $( Property deduction for a metric space.  (Contributed by Mario Carneiro,
+       4-Oct-2015.) $)
+    mspropd $p |- ( ph -> ( K e. MetSp <-> L e. MetSp ) ) $=
+      ( cxms wcel cds cfv cbs cxp cres cmet wa cms eqtr3d eqid xmspropd sqxpeqd
+      reseq2d fveq2d eleq12d anbi12d ctopn isms 3bitr4g ) ACIJZCKLZCMLZULNZOZUL
+      PLZJZQDIJZDKLZDMLZUSNZOZUSPLZJZQCRJDRJAUJUQUPVCABCDEFGHUAAUNVAUOVBAURBBNZ
+      OZUNVAAUKVDOVEUNGAVDUMUKABULEUBUCSAVDUTURABUSFUBUCSAULUSPABULUSEFSUDUEUFU
+      NCUGLZCULVFTULTUNTUHVADUGLZDUSVGTUSTVATUHUI $.
+  $}
+
+  ${
+    setsms.x $e |- ( ph -> X = ( Base ` M ) ) $.
+    setsms.d $e |- ( ph -> D = ( ( dist ` M ) |` ( X X. X ) ) ) $.
+    setsms.k $e |- ( ph -> K =
+      ( M sSet <. ( TopSet ` ndx ) , ( MetOpen ` D ) >. ) ) $.
+    ${
+      setsmsbasg.m $e |- ( ph -> M e. V ) $.
+      setsmsbasg.d $e |- ( ph -> ( MetOpen ` D ) e. W ) $.
+      $( The base set of a constructed metric space.  (Contributed by Mario
+         Carneiro, 28-Aug-2015.) $)
+      setsmsbasg $p |- ( ph -> X = ( Base ` K ) ) $=
+        ( cbs cfv cnx wcel wne c1 c9 tsetndx cts cmopn cop csts co baseslid 1re
+        wceq 1lt9 ltneii basendx neeq12i mpbir cn 9nn eqeltri setsslnid syl2anc
+        fveq2d 3eqtr4d ) ADMNZDOUANZBUBNZUCUDUEZMNZGCMNADEPVCFPVAVEUHKLEVCVBMFD
+        UFOMNZVBQRSQRSUGUIUJVFRVBSUKTULUMVBSUNTUOUPUQURHACVDMJUSUT $.
+
+      $( The distance function of a constructed metric space.  (Contributed by
+         Mario Carneiro, 28-Aug-2015.) $)
+      setsmsdsg $p |- ( ph -> ( dist ` M ) = ( dist ` K ) ) $=
+        ( cds cfv cnx cts wcel wceq wne c9 cmopn cop csts co dsslid cdc 9re 1nn
+        c1 c2 2nn0 9nn0 9lt10 declti dsndx tsetndx neeq12i mpbir cslot tsetslid
+        gtneii cn simpri setsslnid syl2anc fveq2d eqtr4d ) ADMNZDOPNZBUANZUBUCU
+        DZMNZCMNADEQVJFQVHVLRKLEVJVIMFDUEOMNZVISUIUJUFZTSTVNUGUIUJTUHUKULUMUNVA
+        VMVNVITUOUPUQURPVIUSRVIVBQUTVCVDVEACVKMJVFVG $.
+
+      $( The topology of a constructed metric space.  (Contributed by Mario
+         Carneiro, 28-Aug-2015.)  (Revised by Jim Kingdon, 7-May-2023.) $)
+      setsmstsetg $p |- ( ph -> ( MetOpen ` D ) = ( TopSet ` K ) ) $=
+        ( cmopn cfv cnx cts cop csts co wcel tsetslid setsslid syl2anc fveq2d
+        wceq eqtr4d ) ABMNZDOPNUGQRSZPNZCPNADETUGFTUGUIUEKLEUGPFDUAUBUCACUHPJUD
+        UF $.
+    $}
+  $}
+
+  ${
+    $d x y A $.  $d r w x y z D $.  $d r x y J $.  $d x y z R $.  $d w x y S $.
+    $d r y N $.  $d r w x y z P $.  $d z T $.  $d r w x y z X $.
+    mopni.1 $e |- J = ( MetOpen ` D ) $.
+    $( An open set of a metric space includes a ball around each of its points.
```

Comment:
To be precise: "An open set of an extend metric space ..."

But maybe from here on, the term "metric space" can mean also "extended metric space", so no change is necessary.

Reply:
> But maybe from here on, the term "metric space" can mean also "extended metric space", so no change is necessary.

I think that is probably the way to go. If we want to revisit it later we can but it feels like the kind of thing where we might not want to spell out every detail beyond a certain point.